### ISSUE TITLE
feature request: ability to upload the downloaded params to instantiate a model

### ISSUE THREAD

**Issue by steven-murray** [halomod/TheHaloMod-SPA#77]
Would be kinda cool to be able to upload the downloaded model param file (the .toml one) to create a new model. This would also mean that people could take control of their own model persistence.

**Comment by aneuhold** [comment]
Great idea! Just as a note for future implementation, this would require that we version the param files that are downloaded and find out a system for converting old formats to new.

**Comment by steven-murray** [comment]
Yeah, I have thought about that a little bit. I think the way to do this is to use the version of `hmf` and `halomod`. If either has a MAJOR version difference with the one currently running, we should essentially just reject the file as old. We can think about converting between formats later as well, but for now, just making sure we're rejecting the right ones should be enough.
